### PR TITLE
[All] Fix the issue of sending super emoticons and some other normal emoticons

### LINKED ISSUE #367: 消息发送api不支持部分黄豆表情，不支持超级表情
### Lagrange项目

OneBot

### 新需求内容

现在可以接收新的黄豆表情id但是不能发。

比如：收到的id是352，发送352，实际发送的是96。

希望增加对新表情的支持，以及大黄豆表情。

图片是一个复读(直接发送收到的消息)的截图
Screenshot_2024-05-13-11-31-28-664_com tencent mobileqq

### PR DESCRIPTION
**Pull request by pk5ls20**
resolve #367
- [x] Fix the issue of sending super emoticons
- [x] Fix the issue of sending some other normal emoticons & dice & rps emoticons
- [x] Works on my machine

### CODE REVIEW

**Comment by dogdie233** [comment]
@Linwenxuan04

**Review by dogdie233 — APPROVED** [review]

**Comment by pk5ls20** [comment]
还没写完，先别合（

> @Linwenxuan04

File: Lagrange.Core/Internal/Service/Action/JoinEmojiChainService.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+using Lagrange.Core.Internal.Packets.Service.Oidb.Request;
+using Lagrange.Core.Utility.Extension;
+using ProtoBuf;
+
+namespace Lagrange.Core.Internal.Service.Action;
+
+[EventSubscribe(typeof(GroupJoinEmojiChainEvent))]
+[EventSubscribe(typeof(FriendJoinEmojiChainEvent))]
+[Service("OidbSvcTrpcTcp.0x90ee_1")]
+internal class FetchClientKeyService : BaseService<JoinEmojiChainEvent>
```
Comment (Linwenxuan04) [inline Lagrange.Core/Internal/Service/Action/JoinEmojiChainService.cs:14]: Rename Service

File: Lagrange.Core/Internal/Context/Logic/Implementation/OperationLogic.cs [hunk 2]
```diff
@@ -222,7 +223,8 @@ public async Task<string> FetchGroupFSDownload(uint groupUin, string fileId)
         return $"{((GroupFSDownloadEvent)events[0]).FileUrl}{fileId}";
     }
 
-    public async Task<(int, string)> GroupFSMove(uint groupUin, string fileId, string parentDirectory, string targetDirectory)
+    public async Task<(int, string)> GroupFSMove(uint groupUin, string fileId, string parentDirectory,
+        string targetDirectory)
```
Comment (Linwenxuan04) [inline Lagrange.Core/Internal/Context/Logic/Implementation/OperationLogic.cs:227]: Remove useless changes

**Review by Linwenxuan04 — CHANGES_REQUESTED** [review]

**Review by Linwenxuan04 — APPROVED** [review]

**Pull request merged**